install.sh

```diff
@@ -328,6 +328,11 @@ ln -srf "$rprefix/scyllatop/scyllatop.py" "$rprefix/bin/scyllatop"
 
 SBINFILES=$(cd dist/common/scripts/; ls scylla_*setup node_health_check scylla_ec2_check scylla_kernel_check)
 SBINFILES+=" $(cd seastar/scripts; ls seastar-cpu-map.sh)"
+
+cat << EOS > "$rprefix"/scripts/scylla_sysconfdir.py
+SYSCONFDIR="$rsysconfdir"
```

Comment:
$rsysconfdir is not correct. When running install.sh from rpmbuild, it will point to the temporary directory created by rpm, not /etc/sysconfig.

Reply:
Ah, you are right.
But on nonroot mode sysconfdir is located to $rsysconfdir == $rprefix/$sysconfdir, so $rsysconfdir is not incorrect for root mode but it's correct for nonroot mode.
I changed the PR to not modify current code for root mode, fix only nonroot mode.
On new version of the PR, SYSCONFDIR set to:
    root mode:  `SYSCONFDIR="$sysconfdir"`
    nonroot mode: `SYSCONFDIR="$rsysconfdir"`